app/Http/Controllers/Admin/Users/WordFilterController.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * @author    MyBB Group
+ * @version   2.0.0
+ * @package   mybb/core
+ * @license   http://www.mybb.com/licenses/bsd3 BSD-3
+ */
+
+namespace MyBB\Core\Http\Controllers\Admin\Users;
+
+use Illuminate\Http\Request;
+use Illuminate\View\View;
+use DaveJamesMiller\Breadcrumbs\Manager as Breadcrumbs;
+use MyBB\Core\Database\Repositories\WordFilterRepositoryInterface;
+use MyBB\Core\Http\Controllers\Admin\AdminController;
+
+class WordFilterController extends AdminController
+{
+    /**
+     * @var Breadcrumbs
+     */
+    private $breadcrumbs;
+    
+    /**
+     * @var WordFilterRepositoryInterface
+     */
+    private $wordFilterRepository;
+    
+    public function __construct(
+        Breadcrumbs $breadcrumbs,
+        WordFilterRepositoryInterface $wordFilterRepository
+    ) {
+        $this->breadcrumbs = $breadcrumbs;
+        $this->wordFilterRepository = $wordFilterRepository;
+    }
+    
+    public function index() : \Illuminate\View\View
```

Comment:
Personally I prefer to return a response interface in the function signature, in case something changes in the future.